databases/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "vscode-azuredatabases",
+  "author": "Microsoft Corporation",
+  "version": "0.0.1",
+  "description": "Common tools for developing Azure Databases support in Azure App Service extension for VS Code",
+  "tags": [
+    "azure",
+    "vscode"
+  ],
+  "engines": {
+    "vscode": "^1.57.0"
+  },
+  "keywords": [
+    "azure",
+    "vscode"
+  ],
+  "main": "out/src/index.js",
+  "types": "out/src/index.d.ts",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/vscode-azuretools"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/vscode-azuretools/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vscode-azuretools/blob/main/databases/README.md",
+  "scripts": {
+    "build": "tsc -p ./",
+    "prepack": "tsc -p ./",
+    "compile": "tsc -watch -p ./",
+    "lint": "eslint --ext .ts .",
+    "lint-fix": "eslint --ext .ts . --fix",
+    "pretest": "npm run build",
+    "test": "node ./out/test/runTest.js"
+  },
+  "devDependencies": {
+    "@microsoft/eslint-config-azuretools": "^0.1.0",
+    "@types/dotenv": "^6.1.1",
+    "@types/fs-extra": "^8.0.0",
+    "@types/gulp": "^4.0.6",
+    "@types/mocha": "^8.2.2",
+    "@types/node": "^14.0.0",
+    "@types/vscode": "1.57.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "eslint": "^7.19.0",
+    "eslint-plugin-import": "^2.22.1",
+    "glob": "^7.1.6",
+    "gulp": "^4.0.2",
+    "mocha": "^8.3.2",
+    "mocha-junit-reporter": "^2.0.0",
+    "mocha-multi-reporters": "^1.5.1",
+    "ts-node": "^7.0.1",
+    "typescript": "^4.3.5",
+    "vsce": "^1.87.0",
+    "vscode-azureextensiondev": "^0.10.0",
+    "vscode-test": "^1.5.2",
+    "webpack": "^5.28.0",
+    "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "@azure/arm-appservice": "^6.1.0",
+    "@azure/arm-cosmosdb": "^9.1.0",
+    "@azure/arm-postgresql": "^5.0.0",
+    "@azure/arm-postgresql-flexible": "^2.0.0",
+    "@azure/arm-resources": "^4.0.0",
+    "@azure/ms-rest-js": "^2.2.1",
+    "dotenv": "^6.2.0",
+    "fast-xml-parser": "^3.19.0",
+    "fs-extra": "^8.0.0",
+    "keytar": "^7.7.0",
+    "moment": "^2.24.0",
```

Comment:
`moment` is quite large and significantly affects code loading time, I'd recommend switching to `dayjs`.

Reply:
I was checking other repos and some have "moment" installed and included as a dependency but others don't, for example --> VM and Databases (moment is a dependency for a mongo pkg) don't have moment dependency. 

That said, I am not sure how not have "moment" as a direct dependency might impact on the code loading time though, @nturinski @wwlorey  any thoughts?